tests/components/unifi/test_switch.py

```diff
@@ -263,13 +265,13 @@ async def test_switches(hass):
     )
 
     assert len(controller.mock_requests) == 4
-    assert len(hass.states.async_entity_ids("switch")) == 3
+    assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 3
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)